crates/uv-resolver/src/resolution/graph.rs

```diff
@@ -267,8 +273,17 @@ impl ResolutionGraph {
             }
         }
 
+        // Extract the `Requires-Python` range, if provided.
+        // TODO(charlie): Infer the supported Python range from the `Requires-Python` of the
+        // included packages.
```

Comment:
Yeah this seems nice

Reply:
I think it's straightforward but need to try.